docker/2.0.1/py2/Dockerfile.cpu

```diff
@@ -0,0 +1,126 @@
+FROM ubuntu:18.04
+
+LABEL maintainer="Amazon AI"
+
+# prevent stopping by user interaction
+ENV DEBIAN_FRONTEND noninteractive
+ENV DEBCONF_NONINTERACTIVE_SEEN true
+ENV SAGEMAKER_TRAINING_MODULE sagemaker_tensorflow_container.training:main
+
+# Set environment variables for MKL
+# For more about MKL with TensorFlow see:
+# https://www.tensorflow.org/performance/performance_guide#tensorflow_with_intel%C2%AE_mkl_dnn
+ENV KMP_AFFINITY=granularity=fine,compact,1,0 
+ENV KMP_BLOCKTIME=1 
+ENV KMP_SETTINGS=0
+
+ENV PYTHONDONTWRITEBYTECODE=1 
+ENV PYTHONUNBUFFERED=1 
+ENV PYTHONIOENCODING=UTF-8 
+ENV LANG=C.UTF-8 
+ENV LC_ALL=C.UTF-8
+
+ARG FRAMEWORK_SUPPORT_INSTALLABLE=sagemaker_tensorflow_training*.tar.gz
+ARG TF_URL=https://tensorflow-aws.s3-us-west-2.amazonaws.com/2.0.1/AmazonLinux/cpu/final/tensorflow-2.0.1-cp27-cp27mu-manylinux2010_x86_64.whl
```

Comment:
Please confirm that this URL is correct.

It has "2.0.1/AmazonLinux" in it, rather than "Ubuntu" or anything else like that. This container is build using ubuntu as the base container.